pyccel/ast/variable.py

```diff
@@ -32,7 +30,7 @@
     'VariableAddress'
 )
 
-class Variable(Symbol, PyccelAstNode):
+class Variable(PyccelAstNode):
```

Comment:
With the double inheritance fixed, the `__new__` function should no longer be necessary